db/migrate/20170920212535_add_event_id_to_event_streams.rb

```diff
@@ -0,0 +1,5 @@
+class AddEventIdToEventStreams < ActiveRecord::Migration[5.0]
+  def change
+    add_column :event_streams, :event_id, :bigint
```

Comment:
I think we want the reference here to be a string so that it can be used by providers where the event unique reference isn't an int.

As for the name of the column, typically we use either `:ems_ref` and/or `:uid_ems` as the "provider reference".  @Fryguy thoughts/preferences?

Reply:
No problem to change to string.

For the name of the column, I think use `ems` in the name could not be a good choice. We already have others attributes to refer a provider like ems_id, physical_server_id...